libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsTest.kt

```diff
@@ -1557,44 +1410,27 @@ class MapboxRouteLineUtilsTest {
             )
 
         assertEquals("unknown", trafficExpressionData[0].trafficCongestionIdentifier)
-        assertEquals(21, trafficExpressionData.size)
+        assertEquals(22, trafficExpressionData.size)
+        assertEquals(1.0, trafficExpressionData.last().offset, 0.0)
     }
 
     @Test
     fun getAnnotationProvider_whenNumericTrafficSource_matchesDistances() {
         val colorResources = RouteLineColorResources.Builder().build()
-        val routeAsJson = FileUtils.loadJsonFixture(
-            "route-with-congestion-numeric.json"
-        )
-        val route = DirectionsRoute.fromJson(routeAsJson)
+        val route = loadNavigationRoute("route-with-congestion-numeric.json")
 
         val result =
             MapboxRouteLineUtils.getRouteLegTrafficNumericCongestionProvider(colorResources)
 
         assertEquals(
-            route.legs()!!.first().annotation()!!.distance()!!.size,
-            result(route.legs()!!.first()).size
-        )
-    }
-
-    @Test
-    fun getAnnotationProvider_whenNoRouteOptions() {
```

Comment:
`NavigationRoute`s require `RouteOptions` to be present so this test is irrelevant.